### DISCUSSION TITLE
Processo seletivo Front-end

### DISCUSSION THREAD

**Discussion by YuriBrunetto** [frontendbr/forum#1315] [Discussão]
Olá, pessoal!

Estamos desenvolvendo um novo processo seletivo para desenvolvedores front-end na agência em que trabalho, e a primeira parte do processo é um teste que faremos pessoalmente com o entrevistado tendo que resolver probleminhas de front-end de um repositório interno nosso. 

O que gostaria de saber de vocês é o seguinte:

- vocês acham legal esse estilo de processo seletivo? (tendo em vista que as vagas aqui são apenas presenciais);
- quais testezinhos seriam legais para testar o conhecimento de um desenvolvedor front-end;
- para quem já passou por um processo seletivo pessoalmente (como esse que estou propondo), como você acha que daria para não pressionar tanto o entrevistado, para ele se sentir à vontade fazendo o teste;

Desde já, obrigado!

**Comment by daniel-maker** [comment]
Oi Yuri, eu prefiro fazer o teste em casa. Porque quando se esta trabalhando, o melhor é a entrevista ser mais enxuta possível para poder voltar logo ao trabalho.

**Comment by YuriBrunetto** [comment]
@daniel-maker opa! Entendo seu ponto.. Mas nosso teste aqui é para ser breve mesmo, no máximo uns 20~30 minutos, aí acho que é mais tranquilo para o entrevistado 😄

**Comment by felipefialho** [comment]
Issues épicas sobre o assunto: #911 #917

**Comment by pietrobelluno** [comment]
@YuriBrunetto acho que poderia deixar a pessoa mostrar uns sites que ela fez pra vocês avaliarem, as vezes o cara não está afim de fazer o teste ou não se sente empolgado/a vontade de fazer.

Particularmente não acho legal esse tipo de processo por que fica um aspecto de muita pressão, pode ser que o cara saiba muito e não saiba/lembre daquilo que esta sendo pedido no teste e depois ache que foi mal no processo seletivo e tal. Talvez falar que ele pode escrever como ele faria ao invés de fazer um código também é uma boa, para analisar o raciocinio do cara e ele sentir menos pressão.

**Comment by oliveira** [comment]
@YuriBrunetto, eu já participei de alguns processos seletivos e o mais legal, com certeza, foi um em que apresentei um projeto meu para uma galera (meu atual time). Pediram pra explicar como eu pensei no projeto, no porque da forma que fiz, escolha das tecnologias e um pouco sobre padrões. Com esse formato é possível validar os conhecimentos do candidato sem que ele se sinta pressionado.

**Comment by giovannipds** [comment]
@0liveira verdade, pra pessoas que já tem algum portfolio (que é normalmente o que se busca), esta é uma forma interessante.

**Comment by valterbarros** [comment]
Já tive a aportunidade de participar de um processo seletivo aqui na
empresa como entrevistador e tivemos muita duvida em relação a que teste
fazer mas decidimos pegar uma aplicação nossa removemos algumas linhas
importantes e pedimos para o candidato resolver o problema. Assim ficou uma
coisa mais real do dia a dia.
Em sex, 14 de dez de 2018 às 14:42, Giovanni Pires da Silva <
notifications@github.com> escreveu:

**Comment by klarkc** [comment]
Não manda teste técnico para um candidato com perfil ativo GitHub (ou outra coisa), muitas vezes o candidato mantém um perfil ativo justamente para servir de portfolio ou referência, nesses caso a dica do @0liveira é bem legal.

**Comment by felipefialho** [comment]
@0liveira gostei
